Source/FuzzySearch.swift

```diff
@@ -48,11 +68,37 @@ public struct FuzzySearchResult {
 
 public protocol FuzzySearchable {
     var fuzzyStringToMatch: String { get }
+    
+    func fuzzyTokenized() -> FuzzyTokens
```

Comment:
That could be done, would also allow FuzzyTokens to be completely private, as well. Will also take in consideration your comment bellow and commit a few changes